app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -57,6 +49,12 @@ import org.schabi.newpipe.util.Localization
 import java.util.Calendar
 
 class FeedFragment : BaseListFragment<FeedState, Unit>() {
+    private var _feedBinding: FragmentFeedBinding? = null
+    private val feedBinding get() = _feedBinding!!
```

Comment:
Can't you just make them `lateinit` to prevent having to use asserting getters? Though I have little experience in Kotlin, so I may be wrong ;-)

Reply:
`lateinit` variables can't have `null` assigned to them, and the variable needs to be set to null to allow the binding to be garbage collected.

Reply:
Ok, I see